backend/internal/storage/postgres/schema/review.go

```diff
@@ -0,0 +1,81 @@
+package user
+
+import (
+	"context"
+	"platnm/internal/models"
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+
+	"platnm/internal/errs"
+)
+
+type ReviewRepository struct {
+	db *pgxpool.Pool
+}
+
+func (r *ReviewRepository) UserExists(ctx context.Context, id string) (bool, error) {
+
+	rows, err := r.db.Query(ctx, `SELECT * FROM "user" WHERE id = $1`, id)
+	if err != nil {
+		return false, err
+	}
+	defer rows.Close()
+
+	if rows.Next() {
+		return true, nil
+	}
+
+	return false, nil
+}
+
+func (r *ReviewRepository) GetReviewsByUserID(ctx context.Context, id string) ([]*models.Review, error) {
+
+	rows, err := r.db.Query(ctx, "SELECT * FROM review WHERE user_id = $1", id)
+
+	if !rows.Next() {
+		return []*models.Review{}, errs.NotFound("Review", "valid id", id)
```

Comment:
appreciate the thorough error checking! here though, I think it's actually okay and not an error if there's a valid userId with no reviews
```suggestion
		return []*models.Review{}, nil
```

Reply:
Fixed